src/windows/main_window.py

```diff
@@ -1247,6 +1247,8 @@ def actionPreviousMarker_trigger(self, event):
             if selected_clip:
                 all_marker_positions.append(selected_clip.data["position"])
                 all_marker_positions.append(selected_clip.data["position"] + (selected_clip.data["end"] - selected_clip.data["start"]))
+                # add all keyframes. itterate on all properties of Data, look for each points
```

Comment:
Nit: One 't' in 'iterate', 'iterable'.

Also, now that the actual iteration is in a separate function, it might be better to leave out the description of _how_ this is done here, and move it to the function body instead.